src/qtism/data/storage/xml/marshalling/Qti22MarshallerFactory.php

```diff
@@ -41,6 +43,8 @@ public function __construct()
     {
         parent::__construct();
         $this->addMappingEntry('bdo', SimpleInlineMarshaller::class);
+        $this->addMappingEntry(Figure::QTI_CLASS_NAME, Html5FigureMarshaller::class, self::HTML5_NAMESPACE);
+        $this->addMappingEntry(Figcaption::QTI_CLASS_NAME, Html5FigcaptionMarshaller::class, self::HTML5_NAMESPACE);
```

Comment:
aren't they override each others in case as both are figure?

Reply:
No, they shouldn't at all

Reply:
yep now they are not equals